platforms/documentation/samples/src/integTest/groovy/org/gradle/integtests/samples/SamplesApplicationIntegrationTest.groovy

```diff
@@ -104,10 +104,10 @@ application {
         installDir.file("${executableDir}/my-app.bat").text.contains("MODULE_PATH=")
 
         def builder = new ScriptExecuter()
-        builder.workingDir installDir.file(executableDir)
-        builder.executable 'my-app'
-        builder.standardOutput = new ByteArrayOutputStream()
-        builder.errorOutput = new ByteArrayOutputStream()
+        builder.setWorkingDir(installDir.file(executableDir))
+        builder.setExecutable('my-app')
+        builder.setStandardOutput(new ByteArrayOutputStream())
+        builder.setErrorOutput(new ByteArrayOutputStream())
```

Comment:
💅 Use `=`?

Reply:
We can use it yes, even though IntellIj doesn't recognize it correctly. Done